raw/index.rst

```diff
@@ -0,0 +1,128 @@
+loklak
+======
+
+'Lok Lak' is a very tasty Cambodian stir-fry meat dish with a lot of fresh 
+black pepper. I hope not to scare vegetarians with this name, currently I am 
+one as well. And, loklak is also the name of this project that the goal of 
+collecting and indexing social media message data in a peer to peer network 
+to enable users to analyze big data.
+
+loklak server
+-------------
+
+The loklak server is a server application which is able to collect messages 
+from various sources, including twitter. The server contains a search index 
+and a peer-to-peer index sharing interface. All messages are stored in an 
+elasticsearch index. An automatic deployment from the development branch at 
+GitHub is available for tests here https://loklak-server-dev.herokuapp.com
+
+Code: https://github.com/loklak/loklak_server
+
+Project Site: http://api.loklak.org
+
+Documentation: http://dev.loklak.org/server
+
+loklak search
+-------------
+
+loklak search is an Angular.js web application that provides a search 
+interface for end users of loklak data and showcases some of the possibilities 
+to use loklak data for analytics.
+
+Code: https://github.com/fossasia/loklak_search
+
+Project Site: http://loklak.net
+
+Documentation: http://dev.loklak.org/search/
+
+loklak apps
+-----------
+
+loklak apps is a collection of small web apps that make use, analyze and 
+visualize of the data of the loklak server. loklak apps use the loklak api 
+server.
+
+Code: https://github.com/fossasia/apps.loklak.org
+
+Project Site: http://apps.loklak.org
+
+Documentation: http://dev.loklak.org/webapps/
+
+loklak wok
+----------
+
+The loklak wok is an Android application collecting data.
+
+Code: https://github.com/fossasia/loklak_wok
+
+Documentation: http://dev.loklak.org/androidwok/
+
+loklak wok Desktop
+------------------
+
+The loklak wok is a Desktop application collecting data.
+
+Code: https://github.com/loklak/loklak_wok_desktop
+
+Documentation: http://dev.loklak.org/desktopwok/
+
+loklak APIs
+-----------
```

Comment:
I think our primary focus at the moment should be to integrate everything. Such minor changes can me made afterwards.